data/json/itemgroups/stashes.json

```diff
@@ -266,7 +266,13 @@
     "id": "bugout_bad_pack",
     "subtype": "distribution",
     "//": "an item that wouldn't be super useful in a survival situation, recreational items and such.",
-    "entries": [ { "group": "SUS_book", "prob": 50 }, { "group": "games", "prob": 20 }, { "group": "bugout_memento", "prob": 50 } ]
+    "entries": [
+      { "group": "SUS_book", "prob": 50 },
+      { "group": "games", "prob": 20 },
+      { "group": "bugout_memento", "prob": 50 },
+      { "group": "tampon_box_used", "prob": 20 },
+      { "group": "menstrual_pad_box_used", "prob": 20 }
```

Comment:
I think it would more sense to grab a full box instead of a used box for a bug-out bag. I might have a distribution between full and used here.

Reply:
thanks, good catch